test/components/split/splitcomboview.coffee

```diff
@@ -0,0 +1,28 @@
+should = require 'should'
+sinon = require 'sinon'
+shouldSinon = require 'should-sinon'
+KDSplitComboView = require '../../../lib/components/split/splitcomboview'
+KDView = require '../../../lib/core/view'
+
+
+describe 'KDSplitComboView', ->
+  beforeEach ->
+    @sinon = sinon.sandbox.create()
+    @view = new KDView
+      tooltip :
+        title : 'hello'
+
+    @o = {}
+    @o.views = [@view, @view]
+    @instance = new KDSplitComboView @o, {}
```

Comment:
Yes you are right. But because of the lack of defensive codes in most of the classes, inherited class' parent class throw the error by the lack of the constructor options param. Instead of testing it for failed cases, I suggest giving console.warnings for the failed instantiations. Having more tests to cover errors that shouldn't be thrown in the first place, seems like a unnecessary task, in the point of this pull request.